modules/theme-tools/social-menu/icon-functions.php

```diff
@@ -40,8 +43,8 @@ function jetpack_social_menu_get_svg( $args = array() ) {
 
 	// Set defaults.
 	$defaults = array(
-		'icon'        => '',
-		'fallback'    => false,
+		'icon'     => '',
```

Comment:
Nice! GitHub seems confused, but things look good in my editor!

![screen shot 2017-05-05 at 18 16 39](https://cloud.githubusercontent.com/assets/426388/25754307/ff93f3d6-31be-11e7-85cc-589b2d162998.png)

LGTM!